bin/deploy

```diff
@@ -236,12 +236,19 @@ test -f "$LSSTSW/lsst_build/.deployed" || ( # Clone lsst_build
     touch "$LSSTSW/lsst_build/.deployed"
 )
 
-cat <<EOF
-
-Done. Run the following:
-
-    . $LSSTSW/bin/setup.sh
-
-to begin using it.
+echo "Done. Run the following:"
+echo 
+
+if [[ $SHELL =~ bash$ ]]; then
```

Comment:
Oh right, `$SHELL` is not your current shell it's your login shell. Sorry. I had confused myself into thinking it was the current shell (but it clearly isn't given the other discussion regarding `$ZSH_NAME`).